.github/workflows/deploy-previews.yml

```diff
@@ -648,7 +648,7 @@ jobs:
             --format='value(status.url)' \
             --platform managed \
             --region ${{ secrets.REGION }})
-          echo "::set-output name=url::${service_url}"
+          echo "url=${service_url}" >>$GITHUB_OUTPUT
 
```

Comment:
[stop use of save-output(::) as required by GitHub](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/4a7b5ea3add51580ac8ca1ab223083e45cc92aa3)